addons/website/views/snippets/s_cta_mobile.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cta_mobile" name="CTA Mobile">
+    <section class="s_cta_mobile pt80 pb80">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="8">
+                <div class="o_grid_item g-height-6 g-col-lg-12 col-lg-12 o_cc o_cc2 rounded" style="grid-area: 3 / 1 / 9 / 13; z-index: 1; --grid-item-padding-y: 80px; --grid-item-padding-x: 64px;">
+                    <h3>50,000+ companies trust Odoo.</h3>
+                    <p class="lead">Join us and make your company a better place.<br/><br/></p>
+                    <a href="#"><img src="/web/image/website.app_store" class="img img-fluid" alt=""/></a>
+                    <a href="#"><img src="/web/image/website.google_play" class="img img-fluid" alt=""/></a>
```

Comment:
Easy quick win: why not directly putting a link towards the main page of those stores?